x-pack/plugins/ml/public/components/form_label/__snapshots__/form_label.test.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`FormLabel Basic initialization 1`] = `
     className="euiFormLabel"
     id="ml_aria_label_undefined"
   />
-  <Component
+  <JsonTooltip
```

Comment:
https://babeljs.io/docs/en/babel-plugin-transform-react-display-name
which is part of `@babel/preset-react`
supports adding displayName for React 16+ syntax